### PR TITLE
Translation of Quiz into Hindi 

### PR DESCRIPTION
**Pull request by innovation-platform**
I am so sorry for opening another pull request as I faced some issues while pushing. 
I have made the suggested changes.

### CODE REVIEW

**Review by AdityaGarg00 — APPROVED** [review]

File: quiz-app/src/assets/translations/hin.json [hunk 1]
```diff
@@ -0,0 +1,1017 @@
...
+								"isCorrect": "false"
+							},
+							{
+								"answerText": "False",
+								"isCorrect": "true"
+							}
+						]
+					},
+					{
+						"questionText": "IoT डिवाइस हमेशा HTTP पर संचार करते हैं, वेब ऐप्स और अन्य वेब API के समान:",
```
Comment (Anurag-0-1-A) [inline quiz-app/src/assets/translations/hin.json:331]: "questionText": "IoT डिवाइस हमेशा HTTP पर संचार करते हैं, वेब ऐप्स और अन्य वेब API के समान ___:",

हैं is missing in ___.

File: quiz-app/src/assets/translations/hin.json [hunk 2]
```diff
@@ -0,0 +1,1017 @@
...
+								"answerText": "मेजर्मेन्ट्स",
+								"isCorrect": "false"
+							}
+						]
+					},
+					{
+						"questionText": "IoT डिवाइस के ऑफ़लाइन होने पर कमांड का क्या होता है",
+						"answerOptions": [
+							{
+								"answerText": "यह हमेशा बुरा मानना चाहिए जब डिवाइस वापस ऑनलाइन है",
```
Comment (Anurag-0-1-A) [inline quiz-app/src/assets/translations/hin.json:383]: "answerText": "यह हमेशा बुरा मानना चाहिए जब डिवाइस वापस ऑनलाइन है",

I am not really sure about its meaning. Kindly recheck

File: quiz-app/src/assets/translations/hin.json [hunk 3]
```diff
@@ -0,0 +1,1017 @@
...
+							},
+							{
+								"answerText": "गलत",
+								"isCorrect": "false"
+							}
+						]
+					},
+					{
+						"questionText": "तापमान संयंत्र के विकास के लिए विचार करने के लिए कर रहे हैं:",
+						"answerOptions": [
```
[the remark this answers is not in the document]
Comment (Anurag-0-1-A) [reply quiz-app/src/assets/translations/hin.json:479]: Describe the issue.

File: quiz-app/src/assets/translations/hin.json [hunk 4]
```diff
@@ -0,0 +1,1017 @@
...
+					}
+				]
+			},
+			{
+				"id": 14,
+				"title": "सबक 7-स्वचालित संयंत्र पानी: पोस्ट व्याख्यान प्रश्नोत्तरी",
+				"quiz": [
+					{
+						"questionText": "एक रिले किस प्रकार का स्विच है:",
+						"answerOptions": [
```
[the remark this answers is not in the document]
Comment (Anurag-0-1-A) [reply quiz-app/src/assets/translations/hin.json:670]: Should the options be in Hindi or Should they be converted to English for better understanding?

File: quiz-app/src/assets/translations/hin.json [hunk 5]
```diff
@@ -0,0 +1,1017 @@
...
+							{
+								"answerText": "एक उच्च शक्ति डिवाइस को नियंत्रित करने के लिए एक कम शक्ति डिवाइस",
+								"isCorrect": "true"
+							},
+							{
+								"answerText": "कम पावर डिवाइस को नियंत्रित करने के लिए एक उच्च शक्ति उपकरण",
+								"isCorrect": "false"
+							},
+							{
+								"answerText": "धावक एक दौड़ चलाने के लिए उन दोनों के बीच एक डंडा गुजर",
```
Comment (Anurag-0-1-A) [inline quiz-app/src/assets/translations/hin.json:697]: "answerText": "धावक एक दौड़ चलाने के लिए उन दोनों के बीच एक डंडा गुजर" 

Is this meaningful?

File: quiz-app/src/assets/translations/hin.json [hunk 6]
```diff
@@ -0,0 +1,1017 @@
...
+								"answerText": "केवल एक फ़ंक्शन ऐप",
+								"isCorrect": "false"
+							},
+							{
+								"answerText": "एक कार्य ऐप और केवल एक स्टोरेज खाता",
+								"isCorrect": "false"
+							},
+							{
+								"answerText": "एक फ़ंक्शन ऐप, एक स्टोरेज अकाउंट और एप्लिकेशन सेटिंग्स",
+								"isCorrect": "frue"
```
Comment (Anurag-0-1-A) [inline quiz-app/src/assets/translations/hin.json:907]: "answerText": "एक फ़ंक्शन ऐप, एक स्टोरेज अकाउंट और एप्लिकेशन सेटिंग्स",
								"isCorrect": "frue"

change it to true.

**Review by Anurag-0-1-A — CHANGES_REQUESTED** [review]
@AdityaGarg00, @innovation-platform, @EliteWarrior315 
Some changes are required. While everything else is fine :)

**Comment by jimbobbennett** [comment]
@Anurag-0-1-A - are the changes you mention done? Can this be merged?

**Comment by Anurag-0-1-A** [comment]
No @jimbobbennett. 

@innovation-platform Kindly make the mentioned changes, please.

**Pull request closed without merging**